src/util.js

```diff
@@ -212,7 +164,7 @@ function murmurhash2_32_gc(str) {
 // this to produce consistent hashes browsers need to have a consistent
 // ordering of objects. Ben Alpert says that Facebook depends on this, so we
 // can probably depend on this too.
-export const hashObject = (object /* : ObjectMap */) /* : string */ => murmurhash2_32_gc(JSON.stringify(object));
+export const hashObject = (object /* : ObjectMap */) /* : number */ => stringHash(JSON.stringify(object));
```

Comment:
Done! I also profiled this and it seems to be equivalent to not converting to a string.